project-2-part4/assets/scripts/app.js

```diff
@@ -0,0 +1,258 @@
+const ATTACK_VALUE = 10;
```

Comment:
This code should not be a part of Pull Request.

You must create a new branch from `main` . (During development, all feature-branches are created from `development` branch)

Reply:
Yes , it was actually another project in session 2.... You're right.